src/main/java/com/flightstats/hub/cluster/ClusterCacheDao.java

```diff
@@ -163,20 +157,28 @@ public void delete(String name, String basePath) {
         }
     }
 
-    private LastUpdated getLastUpdated(String path) throws Exception {
+    private MostRecentData getMostRecentData(String path) throws Exception {
         Stat stat = new Stat();
         byte[] bytes = curator.getData().storingStatIn(stat).forPath(path);
         Optional<ContentPath> pathOptional = ContentPath.fromUrl(new String(bytes, StandardCharsets.UTF_8));
-        return new LastUpdated(pathOptional.get(), stat.getVersion());
+        return new MostRecentData(pathOptional.get(), stat.getVersion());
     }
 
-    private class LastUpdated {
-        ContentPath key;
-        int version;
+    private class MostRecentData {
+        private ContentPath key;
+        private int version;
 
-        private LastUpdated(ContentPath key, int version) {
+        private MostRecentData(ContentPath key, int version) {
             this.key = key;
             this.version = version;
         }
+
+        public ContentPath getKey() {
```

Comment:
Why not use lombok ```@Value```?

Reply:
We're not using it much in the rest of the hub code.  Would be nice to start though.

Reply:
I've probably told this story before, but apparently the Hub did use lombok mostly in the past. Then Ben (no longer here) had some oddity with his system that wouldn't handle lombok (in IntelliJ I think) that affected no one else and no one else could duplicate. Regardless, Ben and Greg ripped out all lombok. 

Ryan came back from PTO and went "Huh? Where'd all the lombok go?" and since he wasn't leader of the group he just rolled with it.

tl;dr - the reasons for not using Lombok in hub probably aren't valid anymore.

Reply:
One of our hubkeeping notes is to move back to Lombok where possible/convenient because of that...whole...thing.

Reply:
Heh.  That make a lot of sense with how the code reads, actually.  ;-)
I'm all for re-lombokification.